vyxal/elements.py

```diff
@@ -785,7 +785,15 @@ def divisors(lhs, ctx):
             ),
             ctx,
         )
-    return LazyList((lhs[: x + 1] for x in range(len(lhs))))
+
+    @lazylist
+    def gen():
+        temp = []
+        for item in iterable(lhs, ctx=ctx):
+            temp.append(deep_copy(item))
```

Comment:
This is modifying `temp` each time. The same list is being yielded every time instead of a new copy. It should be `yield temp[:]` instead, I think.